.github/CONTRIBUTION.md

```diff
@@ -44,6 +44,14 @@ Run type checking with:
 
     yarn types:check
 
+## Spec checking
+
+Run spec checking with:
```

Comment:
I mean what you are doing is not checking something, the command generates a JSON file for the API spec. And then  it tells you if something is off. If we want to add this to the documentation it should explain what it does, and that you can use it to make sure the props definition is correct

Reply:
That's better. I will try to write them down as you said.